stl/inc/optional

```diff
@@ -109,7 +109,8 @@ struct _Optional_destruct_base<_Ty, false> { // either contains a value of _Ty o
     constexpr _Optional_destruct_base() noexcept : _Dummy{}, _Has_value{false} {} // initialize an empty optional
 
     template <class... _Types>
-    constexpr explicit _Optional_destruct_base(in_place_t, _Types&&... _Args)
+    constexpr explicit _Optional_destruct_base(in_place_t, _Types&&... _Args) noexcept(
+        is_nothrow_constructible_v<_Ty, _Types...>)

```

Comment:
The specialization `_Optional_destruct_base<_Ty, false>` for non-trivial destructors is being updated, but the primary template for trivial destructors was skipped.